fission/src/ui/panels/simulation/WiringPanel.tsx

```diff
@@ -263,14 +286,14 @@ const WiringComponent: React.FC<ConfigComponentProps> = ({ setConfigState, simCo
     const { screenToFlowPosition } = useReactFlow()
     const [nodes, setNodes, onNodesChange] = useNodesState([] as FlowNode[])
     const [edges, setEdges, onEdgesChange] = useEdgesState([] as FlowEdge[])
-    const [_refreshHook, refreshGraph] = useReducer(x => !x, false) // Whenever I use reducers, it's always sketch. -Hunter
+    const [refreshHook, refreshGraph] = useReducer(x => !x, false) // Whenever I use reducers, it's always sketch. -Hunter
```

Comment:
@PepperLola I've been using this as a janky way to refresh the graph. Probably need to re-evaluate.